index.html

```diff
@@ -371,20 +371,23 @@ <h2 class="form-text">
         chat please get in touch.
       </h2>
       <div class="form">
-        <form>
+        <form action="https://formspree.io/f/mzborojb" method="POST">
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- [ ] I would highly recommend using the same fonts that are provided in the Figma template so your UI design is professional.
- Your design:
<img width="168" alt="image" src="https://user-images.githubusercontent.com/77190203/155290849-10af520a-c0c0-4e9d-b7ba-640e6283e35d.png">

- Figma design:
<img width="155" alt="image" src="https://user-images.githubusercontent.com/77190203/155291675-d696d8be-5453-494e-8b7b-07927f70c666.png">